views/index.ejs

```diff
@@ -49,6 +49,25 @@
     <div class="row py-3">
       <!-- Connect options -->
       <div class="col-12 col-md-4">
+        <% if (user) { %>
+          <div class="card">
```

Comment:
would be good to have an id to hook into for the playwright selector. We can use this card for the await on login to ensure that login was successful